Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -587,13 +570,19 @@ private void SetLoadPackageButtonContent()
             if (!string.IsNullOrEmpty(ServerURLTextBox.Text))
             {
                 RQMLoadConfigPackageButton.Content = "Replace";
-                ExampleURLHint.Content = "and click Replace to change RQM Configuration Package";
-
             }
             else
             {
                 RQMLoadConfigPackageButton.Content = "Load";
-                ExampleURLHint.Content = "and Load RQM Configuration Package";
+            }
+
+            if (!string.IsNullOrEmpty(ServerURLTextBox.Text))
```

Comment:
Codacy found an issue: [This condition was just checked on line 570.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5717441)